src/ServicePulse.Host/app/js/views/redirect/view.html

```diff
@@ -48,7 +48,7 @@
                         <div isolate-click class="col-sm-12">
                             <p class="small">
 
-                                <button type="button" class="btn btn-link btn-sm" confirm-title="Confirm action" confirm-click="vm.deleteRedirect(redirect, 'Redirect was deleted', 'Failed to delete redirect')" 
+                                <button type="button" class="btn btn-link btn-sm" confirm-title="Are you sure you want to end the redirect?" confirm-click="vm.deleteRedirect(redirect, 'Redirect was deleted', 'Failed to delete redirect')" 
```

Comment:
Any special notes, concerns or consequences the user should be aware of before deciding to end the redirect?